docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+version: '3.8'
+services:
+  catalog:
+    build: 
+      context: .
+      dockerfile: Dockerfile
+    container_name: sd-catalog-backend
+    image: sd-catalog-backend:latest
+    restart: unless-stopped
+    environment:
+      MONGODB_DATABASE: "sda_catalog"
+      MONGODB_HOSTNAME: mongodb
+    ports:
+      - "5000:5000"
+    volumes:
+      - appdata:/home/www
+    depends_on:
+      - mongodb
+    networks:
+      - frontend
+      - backend
+  mongodb:
+    image: mongo
+    container_name: mongodb
```

Comment:
Also, can you please add the ports for mongodb container:
```
    ports:
      - 27017:27017
```
This will help the users to connect to the DB using GUI tools like MongoDB compass